src/main/java/org/openstreetmap/atlas/checks/database/FlagDatabaseSubCommand.java

```diff
@@ -239,7 +240,8 @@ public void executeFlagStatement(final PreparedStatement sql, final CheckFlag fl
     {
         try
         {
-            sql.setString(1, flag.getIdentifier());
+            sql.setString(1, flag.getUniqueIdentifiers().stream().map(String::toString)
+                    .collect(Collectors.joining(",")));
```

Comment:
The unique identifiers are unsorted so in this case it would be a good idea to call `.sorted()` on the stream before collecting it.